pyrasterframes/src/main/python/docs/reference.pymd

```diff
@@ -63,29 +63,26 @@ Functions to access and change the particulars of a `tile`: its shape and the da
 #### rf_cell_types
 
     Array[String] rf_cell_types()
-    
+
 Print an array of possible cell type names, as below. These names are used in other functions. See @ref:[discussion on NoData](reference.md#masking-and-NoData) for additional details.
 
-```python
```

Comment:
I think you can actually remove the entire section about rf_cell_types and the discussion in the nodata chapter should suffice.

Could you also change on line 61 the link to go to `nodata-handling.md` ?